devito/finite_differences/derivative.py

```diff
@@ -258,10 +267,9 @@ def __call__(self, x0=None, fd_order=None, side=None, method=None, **kwargs):
         if fd_order is not None:
             rkw['fd_order'] = self._filter_dims(_fd_order, as_tuple=True)
 
-        return self._rebuild(**rkw)
+        return self._rebuild(*self.args, **rkw)
```

Comment:
`*self.args` should not be needed; that's what `rargs` is for at the class definition. `_rebuild` should only ever need the new arg/kwargs and inherit everything else from `self` automatically.